widget/view.controller.js

```diff
@@ -85,17 +85,42 @@
     $scope.finishInfoGraphics = widgetBasePath + 'images/finish.png';
     $scope.widgetCSS = widgetBasePath + 'widgetAssets/css/wizard-style.css';
     const fortiGuardConnectorName = 'Fortinet FortiGuard Threat Intelligence';
-
+    $scope.toggleConnectorConfigSettings = { open: true };
+    $scope.toggleParametersSettings = { open: false };
+    $scope.toggleScheduleConfigSettings = { open: false };
     init();
 
     function init() {
+      var pagedCollection = new PagedCollection('keys');
+      var query = {
+        logic: 'AND',
+        limit: 1,
+        filters: [{
+          field: 'key',
+          operator: 'eq',
+          value: 'threat-intel-management-feed-rules-config'
```

Comment:
move this code to the function and can be part of the service.js, if needed

Reply:
it's only used in once in wizard hence not added in service

Reply:
got it. Please move this code to the function

Reply:
can be added in function

Reply:
done